src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/ConfingurationBindingSourceGeneratorTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using SourceGenerators.Tests;
+using Xunit;
+
+namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration.Tests
+{
+#if NETCOREAPP
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/52062", TestPlatforms.Browser)]
```

Comment:
https://github.com/dotnet/runtime/issues/52062 is a closed issue. Assembly.Location returns empty string on browser, single-file and NativeAOT. Not supporting one means other form factors are also unsupported, right? If other form factors are supported by some other mechanics, can we bring all three to the same plan?

Reply:
cc @eerhardt.

Reply:
> https://github.com/dotnet/runtime/issues/52062 is a closed issue.

I'll correct this for this generator and also the logging generator.

Indeed source generators are not supported in these 3 scenarios. cc Roslyn team on whether we have a plan to support any of these eventually. @jaredpar @chsienki @CyrusNajmabadi.